thoth/storages/graph/postgres.py

```diff
@@ -3112,6 +3114,27 @@ def _python_file_digests_sync_analysis_result(self, package_extract_run: Package
                 file=py_file["filepath"],
             )
 
+    def _python_interpreters_sync_analysis_result(
+        self,
+        package_extract_run: PackageExtractRun,
+        document: dict,
+        software_environment: Union[SoftwareEnvironment, ExternalSoftwareEnvironment]
```

Comment:
This parameter is unused now.

Reply:
@fridex , do you want me to delete it?

Reply:
I wonder why linters do not report anything here. Let's keep it then, we will need it in future :) :+1: